inspect/index.ts

```diff
@@ -149,36 +159,43 @@ export function isSpoofedBot(result: ArcjetRuleResult): boolean {
  *
  * @link https://docs.arcjet.com/bot-protection/reference#bot-verification
  */
-export function isVerifiedBot(result: ArcjetRuleResult): boolean {
+export function isVerifiedBot(result: ArcjetRuleResult): boolean | undefined {
   // Use `unknown` argument helpers to guard around the wrong data being passed
   if (isActive(result) && isBotReason(result.reason)) {
     return result.reason.isVerified();
```

Comment:
I know this is already merged, but I was just thinking that maybe we should return undefined if the bot reason is errored as well. What do you think @blaine-arcjet?